Source/WebKit/WebProcess/cocoa/VideoPresentationManager.mm

```diff
@@ -241,19 +242,19 @@ static FloatRect inlineVideoFrame(HTMLVideoElement& element)
 
 void VideoPresentationManager::removeClientForContext(PlaybackSessionContextIdentifier contextId)
 {
-    ASSERT(m_clientCounts.contains(contextId));
+    auto it = m_clientCounts.find(contextId);
```

Comment:
Shouldn't we keep this ASSERT?

Reply:
It keeps happening. So I’m not sure it indicates an issue